product/module.go

```diff
@@ -77,7 +78,8 @@ commerce: {
 			enabled: bool | *false
 			currency: *"€" | !=""
 			if enabled {
-			  jsonTestDataFolder?: string | !=""
+			  jsonTestDataFolder: string | !=""
```

Comment:
Are those fields now really mandatory? Thought that the live search is optional as well as the test data folder in general..